pages/patterns/delete-pattern.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Heading } from '@twilio-paste/core/heading';
+import { Text } from '@twilio-paste/core/text';
+import { Button } from '@twilio-paste/core/button';
+import { Flex } from '@twilio-paste/core/flex';
+import { Paragraph } from '@twilio-paste/core/paragraph';
+import { Table, TBody, Th, THead, Tr, Td } from '@twilio-paste/core/table';
+import { useToaster, Toaster } from '@twilio-paste/core/toast';
+import { DeleteIcon } from '@twilio-paste/icons/cjs/DeleteIcon';
+
+export default function Home() {
+  const [data, setData] = React.useState([
+    {
+      id: 1,
+      friendlyName: 'first service',
+      sid: 4392908903,
+    },
+    {
+      id: 2,
+      friendlyName: 'second service',
+      sid: 8439025738,
+    },
+    {
+      id: 3,
+      friendlyName: 'third service',
+      sid: 8439025703,
+    },
+    {
+      id: 4,
+      friendlyName: 'fourth service',
+      sid: 6239485739,
+    },
+  ]);
+
+  const toaster = useToaster();
+
+  const handleDelete = (service) => {
+    toaster.push({
+      message: 'Service was successfully deleted.',
+      variant: 'success',
+      dismissAfter: 3000,
+    });
+    const updatedServices = data.filter((serviceObj) => serviceObj.id != service.id);
+    setData(updatedServices);
+  };
+
+  return (
+    <>
+      <Toaster {...toaster} />
+      <Heading as="h1" variant="heading10">
+        <Flex hAlignContent="between" vAlignContent="center">
+          Services
+        </Flex>
+        <Paragraph>
+          Services enable you to organize and identify your conversations by use case, or manage them in multiple
+          environments (e.g. dev, stage, prod). This information held within a service is siloed, protecting both your
+          recepients' data.
+        </Paragraph>
+      </Heading>
+
+      <Table>
+        <THead>
+          <Tr>
+            <Th>Friendly Name</Th>
+            <Th>SID</Th>
+            <Th>Actions</Th>
+          </Tr>
+        </THead>
+        <TBody>
+          {data.map((service) => {
+            return (
+              <Tr key={service.id}>
+                <Td>
+                  <Text>{service.friendlyName}</Text>
```

Comment:
i think `as` is a required prop. Check the console logs when running locally

Reply:
I added the `as` prop because Paste docs do list it as required, but I wasn't getting any errors without it and it doesn't change the appearance of the text. Why is it required?

Reply:
Hmm, you should have seen a console log error, if not that's sounds like a bug.

The reason it's required is that it's a generic. `Text` can be any HTML element, but because we don't know _where_ you're using it in your app, we can't make a good  choice of what HTML element it should be. So by forcing you to pick one, we're hoping you, as the consumer of the library, make an informed choice about the HTML semantics for your custom app.